src/app/(website)/(homepage)/_CounterSection/CounterSection.tsx

```diff
@@ -14,8 +14,9 @@ export const CounterSection = () => {
           </span>
         </h2>
         <h2 className="font-semibold text-center text-dark-blue text-2xl">
+        Dostęp do opieki medycznej{' '}
```

Comment:
Is this spacer `{" "}` necessary?

Reply:
I guess it is, alternative for that would be wrapping text (Dostęp do opieki medycznej) in another span so the number is not glued to the text.

Reply:
I would prefer using additional `span` than adding space with empty string.

Reply:
I can change it of course if you both feel like it's the best solution, but what is the reasoning behind it? Is it more readable to you? I'm trying not to add unnecessary nodes as it can slow the app down. Another idea, which I'm not fond of: we could add ml-2 to existing span to create space.

Reply:
I'm not a fan of positioning DOM elements in this way (using empty text spaces)

Reply:
I feel the same as @javac200. In this case you have a few span in the same space and using css to positioning would be more readable to me.